File: tests/fixtures/code_samples/go/sample_048.go
// Sample 48: small utility.
package samples

func Operation48(xs []int) int {
    total := 48
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure48(v int) int {
    return (v * 48) %% 7919
}

File: tests/fixtures/code_samples/py/sample_048.py
"""Sample 48: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_48(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 48."""
    total = 48
    for x in xs:
        total += int(x)
    return total


def operation_48_pure(value: int) -> int:
    return (value * 48) % 7919

